### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by khadijahawa**
finished developing and designing the home page with 4 sections 
it is fully responsive
made a file structure for importing the sections into the home page 
used 2 libraries material UI and react slider 
pulled the dev branch in order to get the updates and solve the conflicts 
//I couldn't include a screenshot for the page fully

### CODE REVIEW

**Review by maqalaqil — APPROVED** [review]
LGTM @khadijahawa great job!

File: src/components/HomeSections/Hero/Hero.jsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
+import { useState, useEffect, useRef } from 'react';
+import { IoIosArrowBack, IoIosArrowForward } from 'react-icons/io';
```
Comment (Iseluin) [inline src/components/HomeSections/Hero/Hero.jsx:2]: Please leave a blank line after those global imports.

File: src/components/HomeSections/Team/Team.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
+import Card from './Card';
+
+function Team() {
+  const memebers = [
+    {
+      id: 1,
+      name: 'Khadija',
+      department: 'Web developer',
```
Comment (Iseluin) [inline src/components/HomeSections/Team/Team.jsx]: Please pay attention to the capital letters. It should be Web Developer.

File: src/components/HomeSections/Team/Team.jsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
+import Card from './Card';
+
+function Team() {
+  const memebers = [
```
Comment (Iseluin) [inline src/components/HomeSections/Team/Team.jsx]: "Members"

File: src/components/HomeSections/Team/Team.jsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    {
+      id: 1,
+      name: 'Khadija',
+      department: 'Web developer',
+      image:
+        'https://ca.slack-edge.com/T02ST1JH0CE-U02TGC9JKS8-3d89cad21cb6-512',
+      // facebook: facebook,
+      // twitter: twitter,
```
Comment (Iseluin) [inline src/components/HomeSections/Team/Team.jsx]: We wont need those so delete them. But add our githubs please.

File: src/components/HomeSections/Team/Team.jsx [hunk 5]
```diff
@@ -0,0 +1,79 @@
+import Card from './Card';
+
+function Team() {
+  const memebers = [
+    {
+      id: 1,
+      name: 'Khadija',
```
Comment (Iseluin) [inline src/components/HomeSections/Team/Team.jsx]: Let's use full names.

File: src/components/HomeSections/Team/Team.jsx [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+      id: 6,
+      name: 'Mustafa',
+      department: 'Web developer',
+      image:
+        'https://ca.slack-edge.com/T02ST1JH0CE-U02T7CZD8J2-10780d1f297d-512',
+
+      // facebook: facebook,
+      // twitter: twitter,
+    },
+  ];
```
Comment (Iseluin) [inline src/components/HomeSections/Team/Team.jsx]: Same as above comments.

File: src/components/HomeSections/Testimonial/Testimonial.jsx [hunk 7]
```diff
@@ -0,0 +1,68 @@
+import Slider from 'react-slick';
+// import { IoIosArrowBack, IoIosArrowForward } from 'react-icons/io';
```
Comment (Iseluin) [inline src/components/HomeSections/Testimonial/Testimonial.jsx]: Delete this if we are not going to use them.

File: src/components/HomeSections/Testimonial/Testomonial.css [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+  display: inline-block;
+}
+.slick-next {
+  right: 3%;
+  z-index: 1;
+  border-radius: 50%;
+  background-color: black;
+  fill: blue;
+  display: inline-block;
+} */
```
Comment (Iseluin) [inline src/components/HomeSections/Testimonial/Testomonial.css]: Why are those commented out? Delete them if you are not going to use them.

File: src/index.js [hunk 9]
```diff
@@ -1,10 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import 'tw-elements';
-
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
+import 'tw-elements';
```
Comment (Iseluin) [inline src/index.js]: Global import to the top please.

**Review by Iseluin — CHANGES_REQUESTED** [review]
1- Please put the images in assets/images. If you want, you can create a Home folder in it to put the images used in the homepage.
2- What is the purpose of the "app" folder?

**Pull request merged**